Scheduler: Update Offset topics (#8124)
DevExpress/devextreme-documentation: api-reference/10 UI Components/dxScheduler/1 Configuration/views/offset.md

---
---
##### shortDescription
Specifies the minute offset applied to configured day durations in the view.

---

> This property shifts the interval between [startDayHour](/api-reference/10%20UI%20Components/dxScheduler/1%20Configuration/views/startDayHour.md '/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#startDayHour') and [endDayHour](/api-reference/10%20UI%20Components/dxScheduler/1%20Configuration/views/endDayHour.md '/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#endDayHour'). The **offset** value can range from `-1440` (24 hours behind) to `1440` (24 hours ahead). 

Reviewer: ```suggestion
This property shifts the interval between [startDayHour](/api-reference/10%20UI%20Components/dxScheduler/1%20Configuration/views/startDayHour.md '/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#startDayHour') and [endDayHour](/api-reference/10%20UI%20Components/dxScheduler/1%20Configuration/views/endDayHour.md '/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#endDayHour'). The **offset** value can be between `-1440` (24 hours before) and `1440` (24 hours ahead). 
```

Author: On second thought, maybe the words "back" and "forward" fit the context better? The property can offset the specified day duration up to 24 hours *back* or *forward*, perhaps not behind/before or ahead. What do you think?
```suggestion
This property shifts the interval between [startDayHour](/api-reference/10%20UI%20Components/dxScheduler/1%20Configuration/views/startDayHour.md '/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#startDayHour') and [endDayHour](/api-reference/10%20UI%20Components/dxScheduler/1%20Configuration/views/endDayHour.md '/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#endDayHour'). The **offset** value can be between `-1440` (24 hours back) and `1440` (24 hours forward).
```

Reviewer: Okay, that is fine
